doc/sr_cpp_walkthrough.rst

```diff
@@ -0,0 +1,99 @@
+***
+C++
+***
+
+
+In this section, examples are presented using the SmartRedis C++
+API to interact with the RedisAI tensor, model, and script
+data types.  Additionally, an example of utilizing the
+SmartRedis ``DataSet`` API is also provided.
+
+.. note::
+    The C++ API examples rely on the ``SSDB`` environment
+    variable being set to the address and port of the Redis database.
+
+.. note::
+    The C++ API examples are written
+    to connect to a Redis cluster database.  Update the
+    ``Client`` constructor call to connect to a Redis non-cluster database.
```

Comment:
Shouldn't this be something along the lines of "if you are using a non-cluster database, update the ``Client`` constructor"? Or is it said somewhere else that the Database is not a cluster?